framework/physics/physics_material/multi_group_xs/multi_group_xs.h

```diff
@@ -32,46 +32,30 @@ class MultiGroupXS : public MaterialProperty
    */
   void ExportToChiXSFile(const std::string& file_name, const double fission_scaling = 1.0) const;
```

Comment:
@andrsd is this name scheduled to be changed in an upcoming PR?

Reply:
Yes.